.github/workflows/dependency_graph.yml

```diff
@@ -0,0 +1,46 @@
+name: Stdlib module dependencies
+
+on: 
+    schedule:
```

Comment:
Shall we add a workflow dispatch trigger too?

Reply:
Done